### PR TITLE
Use sphinx.util.status_iterator if available

### PR DESCRIPTION
**Pull request by jschueller**
My proposition for #112, doesn't require changes in setup.py and allows to get a release before sphinx 1.7.

### CODE REVIEW

**Comment by mgeier** [comment]
Thanks!

The version check (with the added dependency) seems a bit heavy to me.
What about just trying the new one and if it doesn't work, try the old one?

Something like this (untested!):

```python
try:
    status_iterator = sphinx.util.status_iterator
except AttributeError:
    status_iterator = app.status_iterator  # For Sphinx < 1.6
# use status_iterator(...) here
```

**Comment by jschueller** [comment]
Yes, that's even better. I just tested it successfully.

File: nbsphinx.py [hunk 1]
```diff
@@ -1174,8 +1174,13 @@ def html_collect_pages(app):
...
         files.update(file_list)
-    for file in app.status_iterator(files, 'copying linked files... ',
-                                    sphinx.util.console.brown, len(files)):
+    try:
+        status_iterator = sphinx.util.status_iterator
+    except AttributeError:
+        status_iterator = app.status_iterator  # For Sphinx < 1.6
+    for file in status_iterator(files, 'copying linked files... ',
+                                sphinx.util.console.brown,
+                                len(files)):
```
Comment (mgeier) [inline nbsphinx.py]: The line break here isn't necessary anymore.

Comment (jschueller) [inline nbsphinx.py]: ok, fixed

**Comment by mgeier** [comment]
Thanks @jschueller!

**Pull request merged**